lib/dockerfile/instruction-updater.ts

```diff
@@ -0,0 +1,105 @@
+import { DockerfileParser } from "dockerfile-ast";
+import { EOL } from "os";
+import { getDockerfileBaseImageName } from "./instruction-parser";
+
+export { updateDockerfileBaseImageName };
+
+/**
+ * Updates the image name of the last from stage, after resolving all aliases
+ * @param contents Contents of the Dockerfile to update
```

Comment:
we have a typo here? `contents Contents`?

Reply:
`contents` is the name of the parameter, and `Contents` is the first word of the sentence.